src/BugsnagServiceProvider.php

```diff
@@ -269,8 +269,8 @@ protected function setupCallbacks(Client $client, Container $app, array $config)
         if (!isset($config['user']) || $config['user']) {
             $client->registerCallback(new CustomUser(function () use ($app) {
                 if ($user = $app->auth->user()) {
-                    if (is_callable([$user, 'toArray'])) {
-                        return $user->toArray();
+                    if (is_callable([$user, 'attributesToArray'])) {
```

Comment:
Technically yeh, it could be changed there. I don't think any of the out of the box loggers implement `__call`, but that check could be added there, yeh.